### PR TITLE
dev-vcs/svn2cl: Version bump, upstream updated, EAPI 0 -> 7

### PR DESCRIPTION
**Pull request by hlein**
See https://bugs.gentoo.org/698602; I have addressed the feedback in this version.
Also respun to add missing tags.

Signed-off-by: Hank Leininger <hlein@korelogic.com>
Bug: https://bugs.gentoo.org/698602
Package-Manager: Portage-2.3.78, Repoman-2.3.17

### CODE REVIEW

File: dev-vcs/svn2cl/svn2cl-0.14.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Create a GNU-style ChangeLog from subversion's svn log --xml output"
+HOMEPAGE="https://arthurdejong.org/svn2cl/"
+SRC_URI="${HOMEPAGE}/${PN}/${P}.tar.gz"
```
Comment (mgorny) [inline dev-vcs/svn2cl/svn2cl-0.14.ebuild]: Don't use HOMEPAGE in SRC_URI, HOMEPAGE is meant to be multi-value.

File: dev-vcs/svn2cl/svn2cl-0.14.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+
+RDEPEND="dev-libs/libxslt
+	dev-vcs/subversion"
+
+src_prepare() {
+	# the wrapper script looks for the xsl files in the
+	# same directory as the script.
+	eapply -p0 "${FILESDIR}"/0.9-wrapper.patch
```
Comment (mgorny) [inline dev-vcs/svn2cl/svn2cl-0.14.ebuild]: Could you update the patch to apply with `-p1` while at it? Should be sufficient to add `a/` `b/` prefixes before paths.

File: dev-vcs/svn2cl/svn2cl-0.14.ebuild [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+src_install() {
+	newbin svn2cl.sh svn2cl
+	insinto /usr/share/svn2cl
+	doins svn2cl.xsl svn2html.xsl
+	dodoc README NEWS TODO ChangeLog authors.xml svn2html.css
+	doman svn2cl.1
+}
+
+pkg_postinst() {
+	einfo "You can find samples of svn2html.css and authors.xml in"
```
Comment (mgorny) [inline dev-vcs/svn2cl/svn2cl-0.14.ebuild]: That sounds like `elog` candidate.

**Comment by hlein** [comment]
> Would you be interested in becoming a maintainer for this package?
> 
> https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml

Sure, I will give it a shot.  When I address your other feedback and update metadata.xml, should I simply update the branch that this PR is against?  Or should I delete and recreate the branch and PR with one clean new patchset?

File: dev-vcs/svn2cl/svn2cl-0.14.ebuild (hunk 1 above)
Comment (hlein) [reply dev-vcs/svn2cl/svn2cl-0.14.ebuild]: Ah, fixed.

File: dev-vcs/svn2cl/svn2cl-0.14.ebuild (hunk 2 above)
Comment (hlein) [reply dev-vcs/svn2cl/svn2cl-0.14.ebuild]: Done.

File: dev-vcs/svn2cl/svn2cl-0.14.ebuild (hunk 3 above)
Comment (hlein) [reply dev-vcs/svn2cl/svn2cl-0.14.ebuild]: Good call, done.

**Comment by mgorny** [comment]
```
>>> Downloading 'https://arthurdejong.org/svn2cl/svn2cl/svn2cl-0.14.tar.gz'
--2019-11-03 08:30:02--  https://arthurdejong.org/svn2cl/svn2cl/svn2cl-0.14.tar.gz
Resolving arthurdejong.org (arthurdejong.org)... 185.107.212.88
Connecting to arthurdejong.org (arthurdejong.org)|185.107.212.88|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-11-03 08:30:03 ERROR 404: Not Found.

!!! Couldn't download 'svn2cl-0.14.tar.gz'. Aborting.
```

Also feel free to add a commit removing it from package.mask.

**Comment by hlein** [comment]
> 2019-11-03 08:30:03 ERROR 404: Not Found.
> 
> !!! Couldn't download 'svn2cl-0.14.tar.gz'. Aborting.
> ```
> 
> Also feel free to add a commit removing it from package.mask.

Doh.  Fixed, and done.  Thanks!

**Comment by hlein** [comment]
> ## Pull request CI report
> _Report generated at_: 2019-11-03 18:39 UTC
> _Newest commit scanned_: [6bcbab6](https://github.com/gentoo/gentoo/commit/6bcbab650f20a73273c4c1e2bdef0f0832c87fbf)
> _Status_:  **broken**
> 
> There are too many broken packages to determine whether the breakages were added by the pull request. If in doubt, please rebase.
> 
> Issues:https://qa-reports.gentoo.org/output/gentoo-ci/14c5ae0/output.html#app-admin/logstash-bin
> https://qa-reports.gentoo.org/output/gentoo-ci/14c5ae0/output.html#app-crypt/veracrypt

Rebased to hopefully get past this.

**Comment by mgorny** [comment]
I'm sorry but it seems that Aaron Bauman has already ported the existing version to EAPI 7, and e.g. updated the patch. Could you rebase your work on top of that?

**Comment by hlein** [comment]
> I'm sorry but it seems that Aaron Bauman has already ported the existing version to EAPI 7, and e.g. updated the patch. Could you rebase your work on top of that?

Done; I trashed this branch and started over, please see https://github.com/gentoo/gentoo/pull/13717

**Pull request closed without merging**